### PR TITLE
[SPARK 9902] [MLlib] Add Java and Python examples to user guide for 1-sample KS test

### PR DESCRIPTION
**Pull request by josepablocam**
added doc examples for python.

### CODE REVIEW

**Comment by mengxr** [comment]
add to whitelist

**Comment by mengxr** [comment]
ok to test

File: docs/mllib-statistics.md [hunk 1]
```diff
@@ -438,22 +438,42 @@ run a 1-sample, 2-sided Kolmogorov-Smirnov test. The following example demonstra
...
+<div data-lang="python" markdown="1">
+[`Statistics`](api/python/index.html#org.apache.spark.mllib.stat.Statistics$) provides methods to
+run a 1-sample, 2-sided Kolmogorov-Smirnov test. The following example demonstrates how to run
+and interpret the hypothesis tests.
+
+{% highlight python %}
+from pyspark.mllib.stat import Statistics
+
+localData = [1.0, 2.0, ... ] #a list of doubles
+parallelData =  sc.parallelize(localData) # an RDD of Double
```
Comment (mengxr) [inline docs/mllib-statistics.md]: `data = sc.parallelize([1.0, 2.0, ...])` should be sufficient.

**Comment by mengxr** [comment]
@josepablocam For Java example, you can check the code here: https://github.com/apache/spark/blob/master/mllib/src/test/java/org/apache/spark/mllib/stat/JavaStatisticsSuite.java#L63.

**Comment by josepablocam** [comment]
@mengxr added Java example now, thanks!

File: docs/mllib-statistics.md [hunk 2]
```diff
@@ -438,22 +438,63 @@ run a 1-sample, 2-sided Kolmogorov-Smirnov test. The following example demonstra
...
+
+{% highlight java %}
+import com.google.common.collect.Lists;
+import org.apache.spark.api.java.JavaDoubleRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.mllib.stat.Statistics;
+import org.apache.spark.mllib.stat.test.KolmogorovSmirnovTestResult;
+
+JavaSparkContext jsc = ...
+JavaDoubleRDD data = jsc.parallelizeDoubles(Lists.newArrayList(0.2, 1.0, ...));
```
Comment (mengxr) [inline docs/mllib-statistics.md]: minor: We can use `Arrays.asList` to replace `Lists.newArrayList`. The former is from Java core, which doesn't require additional dependencies. Then we can replace `import com.google.common.collect.Lists;` by `import java.util.Arrays;`. Btw, please insert an empty line between Java imports and Spark imports.

File: docs/mllib-statistics.md [hunk 3]
```diff
@@ -438,22 +438,63 @@ run a 1-sample, 2-sided Kolmogorov-Smirnov test. The following example demonstra
...
+KolmogorovSmirnovTestResult testResult = Statistics.kolmogorovSmirnovTest(data, "norm", 0.0, 1.0);
+// summary of the test including the p-value, test statistic,
+// and null hypothesis
+// if our p-value indicates significance, we can reject the null hypothesis
+System.out.println(testResult1);
+{% endhighlight %}
+</div>
+
+<div data-lang="python" markdown="1">
+[`Statistics`](api/python/index.html#pyspark.mllib.stat.Statistics$) provides methods to
```
Comment (mengxr) [inline docs/mllib-statistics.md]: The link should be `api/python/pyspark.mllib.html#pyspark.mllib.stat.Statistics`.

**Comment by mengxr** [comment]
LGTM except minor inline comments.

**Comment by josepablocam** [comment]
@mengxr thanks for the feedback. Fixed

**Comment by mengxr** [comment]
LGTM. Merged into master and branch-1.5. Thanks!

**Pull request closed without merging**